assets/js/components/help/HelpMenuLink.js

```diff
@@ -33,18 +33,25 @@ import Link from '@/js/components/Link';
 import { trackEvent } from '@/js/util';
 import useViewContext from '@/js/hooks/useViewContext';
 
-function HelpMenuLink( { children, href, gaEventLabel } ) {
+function HelpMenuLink( {
+	children,
+	href,
+	gaEventLabel,
+	onClick: onClickCallback,
+	icon,
+} ) {
 	const viewContext = useViewContext();
 
 	const onClick = useCallback( async () => {
+		onClickCallback?.();
```

Comment:
Let's rename the callback rather than aliasing the prop name.

```suggestion
	onClick,
	icon,
} ) {
	const viewContext = useViewContext();

	const handleClick = useCallback( async () => {
		onClick?.();

```